examples/compute/create_instance.go

```diff
@@ -0,0 +1,83 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"time"
+
+	triton "github.com/joyent/triton-go"
+	"github.com/joyent/triton-go/authentication"
+	"github.com/joyent/triton-go/compute"
+)
+
+func main() {
+	keyID := os.Getenv("SDC_KEY_ID")
+	accountName := os.Getenv("SDC_ACCOUNT")
+	signer, err := authentication.NewSSHAgentSigner(keyID, accountName)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	config := &triton.ClientConfig{
+		TritonURL:   os.Getenv("SDC_URL"),
+		AccountName: accountName,
+		Signers:     []authentication.Signer{signer},
+	}
+	c, err := compute.NewClient(config)
+	if err != nil {
+		log.Fatalf("NewClient(): %s", err)
+	}
+
+	// Create a new instance using our input attributes...
+	// https://github.com/joyent/triton-go/blob/master/compute/instances.go#L206
+	createInput := &compute.CreateInstanceInput{
+		Name:     "go-test1",
+		Package:  "g4-highcpu-512M",
+		Image:    "1f32508c-e6e9-11e6-bc05-8fea9e979940",
```

Comment:
What datacenter is this instance out of?  It may be worth noting the image ID is tied to a DC atm.